index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Rocket Man</title>
+    <link rel="stylesheet" href="css/game.css" type="text/css">
```

Comment:
no link to normalize? you have included it in the branch but not linked it?

Reply:
Fixed in 7d7f458